ember-headless-form/src/components/-private/control/textarea.hbs

```diff
@@ -0,0 +1,5 @@
+<textarea
+  id={{@fieldId}}
+  ...attributes
```

Comment:
Question, curious: I see in other component systems there are options to `resize` vertically and horizontally, is this something we'd think of doing at this level or above the `headless` layer?

Reply:
Hm, I don't really have prior experience with this, but don't browsers add this ootb now? Like even with a plain `<textarea>` in Chrome there is this dragging "button" on the lower right corner.

But even when there were some more custom solutions to this, I would think that is not something we need to do at this headless level.

Reply:
agreed -- resize is automatic -- we can control horiz / vert-only resizing if we want to, but I don't think we need to here.